.eslintrc.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
+  env: {
+    commonjs: true,
+    es6: true,
+    node: true,
+  },
+  extends: [
+    'airbnb-base',
```

Comment:
We don't need this kind of hostility in our life.  in the words of Dan Abormov - extend prettier [config](https://prettier.io/docs/en/integrating-with-linters.html). Also, remember to Add add a prettier config file.

Reply:
Done.